alpha/alpha-server/pom.xml

```diff
@@ -258,5 +258,15 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>spring-cloud</id>
```

Comment:
My suggestion is using spring-cloud-consul by default. We can fix it later.

Reply:
rename spring-cloud-eureka